src/WebDriverProxy.ts

```diff
@@ -0,0 +1,297 @@
+import * as express from 'express';
+import * as request from 'request';
```

Comment:
Could we use `request` from `@theintern/common` since that's already a dep?

Reply:
I'll have to refactor my proxying function.

Reply:
I refactored the code to use `request` from `@theintern/common`.  I created a PR, https://github.com/theintern/common/pull/4, for a small change to common.